### PR TITLE
Package Test.Utility as a package but don't push it from VSTS

### PR DESCRIPTION
**Pull request by joelverhagen**
This allows server-side to use the Test.Utility package. I plan on manually pushing it to the nuget-build MyGet feed.

### CODE REVIEW

File: scripts/cibuild/PublishArtifactsFromVsts.ps1 [hunk 1]
```diff
@@ -67,7 +67,7 @@ if(Test-Path $NupkgsDir)
 {
     Write-Output "Publishing nupkgs from '$Nupkgs'"
     # Push all nupkgs to the nuget-build feed on myget.
-    Get-Item "$NupkgsDir\*.nupkg"  | Push-ToMyGet -ApiKey $NuGetBuildFeedApiKey -BuildFeed $NuGetBuildFeedUrl -SymbolSource $NuGetBuildSymbolsFeedUrl
+    Get-Item "$NupkgsDir\*.nupkg" -Exclude "Test.Utility.*.nupkg" | Push-ToMyGet -ApiKey $NuGetBuildFeedApiKey -BuildFeed $NuGetBuildFeedUrl -SymbolSource $NuGetBuildSymbolsFeedUrl
```
Comment (joelverhagen) [inline scripts/cibuild/PublishArtifactsFromVsts.ps1]: Thoughts on making the exclude `*Test*.nupkg` to be more future proof?

Comment (rohit21agrawal) [reply scripts/cibuild/PublishArtifactsFromVsts.ps1]: i'd say do Test*

Comment (joelverhagen) [reply scripts/cibuild/PublishArtifactsFromVsts.ps1]: I was thinking that maybe sometime `Test.Utility` would be renamed to `NuGet.Test.Utility`.

Comment (joelverhagen) [reply scripts/cibuild/PublishArtifactsFromVsts.ps1]: Fixed.

**Review by rohit21agrawal — APPROVED** [review]

**Review by emgarten — APPROVED** [review]

**Review by jainaashish — APPROVED** [review]

**Pull request merged**